apps/backend/proto/kandev/plugin/v1/plugin.proto

```
@@ -15,13 +15,55 @@ service Plugin {
+// service methods that publish task.* events (never repository.TaskRepository).
 service Host {
   rpc GetState(GetStateRequest) returns (GetStateResponse);
   rpc SetState(SetStateRequest) returns (SetStateResponse);
   rpc DeleteState(DeleteStateRequest) returns (DeleteStateResponse);
   rpc ListState(ListStateRequest) returns (ListStateResponse);
   rpc RevealSecret(RevealSecretRequest) returns (RevealSecretResponse);
   rpc EmitEvent(EmitEventRequest) returns (EmitEventResponse);
+
+  // Reads — capability api_read:<resource>
+  rpc ListTasks(ListTasksRequest) returns (ListTasksResponse);
+  rpc GetTask(GetTaskRequest) returns (Task);
```

Buf (1.71.0)

[error] "kandev.plugin.v1.Task" is used as the request or response type for multiple RPCs.

(RPC_REQUEST_RESPONSE_UNIQUE)